test/rules/migration/no-deprecated-properties.test.ts

```diff
@@ -31,7 +31,7 @@ export default class EnvCreateScratch extends SfCommand<ScratchCreateResponse> {
       filename: path.normalize('foo.ts'),
       code: `
 export default class EnvCreateScratch extends SfCommand<string> {
-  public static readonly requiresUsername = true;
+  public static readonly args = true;
```

Comment:
`no-deprecated-properties` removes properties it matches against.  The new rules (`no-username-properties`) does a smarter thing (import the flag, add the `target-org`, etc).

Since `no-deprecated-properties` no longer covers `requiresUsername` I needed a different test case of something that we do still want to just remove.